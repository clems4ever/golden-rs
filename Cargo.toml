[workspace]

resolver = "2"

members = [
    "datadir",
    "golden-files"
]
