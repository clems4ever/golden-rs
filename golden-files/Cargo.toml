[package]
name = "golden-files"
version = "0.1.0"
edition = "2024"
authors = ["Cl√©ment Michaud"]
description = "Utility function to dump and read data into and from golden files."
license = "MIT"

[dependencies]
serde = "1.0"
serde_json = "1.0"
