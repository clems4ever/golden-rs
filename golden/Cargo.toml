[package]
name = "golden"
version = "0.1.0"
edition = "2024"
authors = ["ClÃ©ment Michaud"]
description = "Create a datadir available during the test to dump golden files."
license = "MIT"

[dependencies]
serde = "1.0"
serde_json = "1.0"
